Fix Bit String to Frame producing 8x too many output items.
See https://github.com/Mictronics/ais-simulator/issues/9

--- a/gr-ais_simulator/lib/bitstring_to_frame_impl.cc
+++ b/gr-ais_simulator/lib/bitstring_to_frame_impl.cc
@@ -438,7 +438,7 @@

                 // output
                 memcpy(out, byte_frame, len_frame_real / 8);
-                noutput_items = len_frame_real;
+                noutput_items = len_frame_real/8;
             }
             else
             {
